commit b0369ebbca69
Author: Gijs Kruitbosch <gijskruitbosch@gmail.com>
Date:   Thu Sep 28 12:00:22 2017 +0100

    Bug 1399939 - Switch to highly restrictive profile for IDN. r=dveditz, r=jfkthame, a=sledru
    
    MozReview-Commit-ID: E5LRuQVfIZP
    
    --HG--
    extra : source : e1c1ebf60808e5e94b9a03d8cb577990ac0b9ca1
---
 modules/libpref/init/all.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git modules/libpref/init/all.js modules/libpref/init/all.js
index c74659f8d4a5..c796df0fef1d 100644
--- modules/libpref/init/all.js
+++ modules/libpref/init/all.js
@@ -1913,7 +1913,7 @@ pref("network.IDN_show_punycode", false);
 // IDN-safe. Otherwise, they're treated as unsafe and punycode will be used
 // for displaying them in the UI (e.g. URL bar), unless they conform to one of
 // the profiles specified in
-// http://www.unicode.org/reports/tr36/proposed.html#Security_Levels_and_Alerts
+// https://www.unicode.org/reports/tr39/#Restriction_Level_Detection
 // If "network.IDN.restriction_profile" is "high", the Highly Restrictive
 // profile is used.
 // If "network.IDN.restriction_profile" is "moderate", the Moderately
@@ -1922,7 +1922,7 @@ pref("network.IDN_show_punycode", false);
 // Note that these preferences are referred to ONLY when
 // "network.IDN_show_punycode" is false. In other words, all IDNs will be shown
 // in punycode if "network.IDN_show_punycode" is true.
-pref("network.IDN.restriction_profile", "moderate");
+pref("network.IDN.restriction_profile", "high");
 pref("network.IDN.use_whitelist", false);
 
 // ccTLDs
